genesyscloud/knowledge_document_variation/resource_genesyscloud_knowledge_document_variation.go

```diff
@@ -232,7 +232,7 @@ func readKnowledgeDocumentVariation(ctx context.Context, d *schema.ResourceData,
 		d.SetId(newId)
 
 		_ = d.Set("knowledge_base_id", *knowledgeDocumentVariation.Document.KnowledgeBase.Id)
-		_ = d.Set("knowledge_document_id", ids.documentID)
+		_ = d.Set("knowledge_document_id", *knowledgeDocumentVariation.Document.Id)
```

Comment:
Do we need to do some nil checks here.  Are we at risk of of ever having a nil pointer exception

Reply:
done